src/commands/createApiCenterService.ts

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import * as vscode from 'vscode';
+import { ApiCenterService } from "../azure/ApiCenter/ApiCenterService";
+import { ResourceGroup } from "../azure/ApiCenter/contracts";
+import { ResourceGraphService } from "../azure/ResourceGraph/ResourceGraphService";
+import { SubscriptionTreeItem } from "../tree/SubscriptionTreeItem";
+import { UiStrings } from "../uiStrings";
+import { GeneralUtils } from "../utils/generalUtils";
+export namespace AzureApiCenterService {
+    export async function createApiCenterService(context: IActionContext, node?: SubscriptionTreeItem): Promise<void> {
+        if (!node) {
```

Comment:
Better to support running this command in Comman Palette

Reply:
supported